advanced_functionality/distributed_tensorflow_mask_rcnn/README.md

```diff
@@ -0,0 +1,132 @@
+# Distributed TensorFlow training using Amazon SageMaker
+
+## Prerequisites
+1. [Create and activate an AWS Account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)
+
+2. [Manage your SageMaker service limits](https://aws.amazon.com/premiumsupport/knowledge-center/manage-service-limits/). You will need a minimum limit of 2 ```ml.p3.16xlarge``` and 2 ```ml.p3.24xlarge``` instance types, but a service limit of 4 for each instance type is recommended. Keep in mind that the service limit is specific to each AWS region. We recommend using ```us-west-2``` region for this tutorial.
```

Comment:
Typo /ml.p3.24xlarge/ml.p3**dn**.24xlarge/

Reply:
Fixed typo. Also added ml.p3dn.24xlarge as an option in the notebooks.